internal/models/account_model.go

```diff
@@ -9,5 +13,5 @@ type Account struct {
 	OrgID string `db:"org_id" json:"org_id"`
 
 	// EBS account number. Can be NULL but not blank.
-	AccountNumber *string `db:"account_number" json:"account_number"`
+	AccountNumber db.NullString `db:"account_number" json:"account_number"`
```

Comment:
Can you try this:

```suggestion
	AccountNumber db.NullString `db:"account_number" json:"account_number,string"`
```

Then regenerate the OpenAPI to see if it picks up the string type here. Then it should work, but my expectations are low :)

Reply:
This does not help either.

Reply:
Damn, okay, apologies for this extra work I really hoped it will work out.

Please remove embedding of models.Account in AccountPayload and simply copy ID/OrgID there, use `AccountNumber *string` in the payload type and `AccountNumber sql.NullString` in the model type. Move json go struct tags from the model to the payload as model will no longer be serializable. And finally, update NewAccountResponse and NewAccountListResponse functions to convert between the two types.

Converting sql.NullString to *string will be reused, so create payloads/null_convert.go file with a helper function that can be used in other payloads as well.